db/migrate/20240110152635_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.string :text
```

Comment:
- [ ] This should be `t.text`, not `t.string`. Kindly fix that. The `text` type has more storage limit compared to `string`

Reply:
Please I want to know if the type `text` is valid in rails